si_ps_conv_fastcorrsApp/src/strn2curr.py

```diff
@@ -3,22 +3,28 @@
 from siriuspy.search import MASearch
 from devsup.db import getRecord
 
+
 class Converter(object):
     raw = True
+
     def __init__(self, rec, args):
-        parsed_args = args.split()
-        psname = parsed_args[0][:-1]
+        test, sec, dip, ori, base = args.split()
+        psname = f"SI-{sec}{dip}:PS-{ori}"
         maname = MASearch.conv_psname_2_psmaname(psname)
         self.norm = NormalizerFactory.create(maname)
 
-        self.dipole_strength = 3
-        self.base_record = getRecord("copy-" + parsed_args[0]+parsed_args[1])
+        self.dipole_strength = getRecord(
+            f"copy-SI-{sec}:Fam:PS-B1B2-1:EnergyRef-Mon")
+        self.base_record = getRecord(f"{test}out-{psname}:{base}")
 
     def process(self, rec, reason):
-        self.base_record.VAL = norm.conv_strength_2_current(rec.VAL, strengths_dipole=self.dipole_strehgth)
+        self.base_record.VAL = self.norm.conv_strength_2_current(
+            rec.VAL, strengths_dipole=self.dipole_strength.VAL)
+        self.base_record.scan(sync=True)
```

Comment:
Do you think it could work to remove this scan, and use a `FLNK` in the record instead?